app/src/main/res/values/strings.xml

```diff
@@ -188,22 +188,22 @@
     <string name="bsct_conn_rule_explanation">
         &lt;u>#1&lt;/u>: Blocks an app from connecting to the Internet.
         \n &lt;u>#2&lt;/u>: Blocks any app from connecting to a particular IP.
-        \n &lt;u>#3&lt;/u>: Blocks all connections when device locked (Universal Firewall).
-        \n &lt;u>#4&lt;/u>: Blocks any app not in use (Universal Firewall).
-        \n &lt;u>#5&lt;/u>: Blocks connections when source app unknown (Universal Firewall).
-        \n &lt;u>#6&lt;/u>: Blocks all UDP traffic except DNS (Universal Firewall).
-        \n &lt;u>#7&lt;/u>: Block connections when DNS is bypassed (Universal Firewall).
+        \n &lt;u>#3&lt;/u>: Blocks all connections when device locked (Universal firewall).
+        \n &lt;u>#4&lt;/u>: Blocks any app not in use (Universal firewall).
+        \n &lt;u>#5&lt;/u>: Blocks connections when source app unknown (Universal firewall).
+        \n &lt;u>#6&lt;/u>: Blocks all UDP traffic except DNS (Universal firewall).
+        \n &lt;u>#7&lt;/u>: Block connections when DNS is bypassed (Universal firewall).
         </string>
 
-    <string name="private_dns_toast">Private DNS has been set.</string>
-    <string name="always_on_dialog_heading">Disable Always-on VPN</string>
-    <string name="always_on_dialog">\"Always-On VPN\" is enabled on this device. To start Rethink, please go to Settings app and disable \"Always-on VPN\".</string>
+    <string name="private_dns_toast">Private DNS set.</string>
+    <string name="always_on_dialog_heading">Action needed</string>
+    <string name="always_on_dialog">\"Always-On VPN\" is turned on for this device. To start Rethink, please go to the Android Settings and turn off \"Always-on VPN\".</string>
```

Comment:
This is a problem, and it should be turned off? Why?
To start _using_ rethink?

Reply:
Always-on VPN for someother app is turned on. This dialog is shown when the user has granted vpn permission to Rethink, and taps on "Start".